mne_bids/path.py

```diff
@@ -233,24 +270,46 @@ def update(self, **entities):
             # Ensure that run is a string
             entities['run'] = '{:02}'.format(run)
 
+        # ensure extension starts with a '.'
+        ext = entities.get('extension')
+        if ext is not None:
```

Comment:
Why do you use `.get()` here? The key should always exist in the dict, no?

Reply:
`entities` is a `**kwargs`, so when people do `path.update(subject='01')`, then 'session' and `kind` wouldn't be inside `entities`.

Reply:
Oh, I see, thanks for the explanation!